test/integration/framework/etcd.go

```diff
@@ -98,7 +101,22 @@ func init() {
 }
 
 // RunCustomEtcd starts a custom etcd instance for test purposes.
-func RunCustomEtcd(dataDir string, customFlags []string, output io.Writer) (url string, stopFn func(), err error) {
+func RunCustomEtcd(logger klog.Logger, dataDir string, customFlags []string) (url string, stopFn func(), err error) {
+	for i := 0; i < 100; i++ {
```

Comment:
we need to be very conservative opening network sockets, if something is wrong a few times then we need to fail because that is sympton something is wrong, at max use 5

Reply:
Obsoleted, using Unix Domain sockets now.